clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractStickyAssignor.java

```diff
@@ -579,7 +579,7 @@ protected List<TopicPartition> getAllTopicPartitions(List<String> sortedAllTopic
     private class ConstrainedAssignmentBuilder extends AbstractAssignmentBuilder {
 
         private final Set<TopicPartition> partitionsWithMultiplePreviousOwners;
-        private final Set<TopicPartition> allRevokedPartitions;
+        private final Map<TopicPartition, String> mayRevokedPartitions;
```

Comment:
nit: this name is a bit confusing, how about `maybeRevokedPartitions`?

Reply:
modified, plz check again.

Reply:
maybeRevokedPartitions sounds very comprehensive and short. It is indeed a nice choice of words.

More options for who do not prefer 'maybe' may be: possiblyRevokedPartitions, perhapsRevokedPartitions.

Reply:
> More options for who do not prefer 'maybe' may be: possiblyRevokedPartitions, perhapsRevokedPartitions.

I actually considered both of those as well -- ultimately went with "maybe" since the variables in this class are all way too long already, but I'm happy with any of these choices 🙂